### PR TITLE
fix(#490): add retries for e2e test TestGetProcessInstanceElementStat…

### LINKED ISSUE #490: Race condition in element statistics – parallel multi-instance body token count is sometimes incomplete
### Description

Flaky test – `TestGetProcessInstanceElementStatisticsMultiInstance/parallel_multi-instance_shows_body_tokens_not_scope_token` intermittently fails.

- **Expected:** Element statistics return `2` active body tokens for a parallel multi-instance activity.
- **Actual:** Sometimes returns `1` instead of `2`.

The test passes most of the time but fails sporadically, which points to a likely **race condition** – the second child token may not yet be fully created/registered when the statistics query executes.

### Steps to Reproduce

1. Run tests in ` TestGetProcessInstanceElementStatisticsMultiInstance`
2. Repeat multiple times (or run with `-count=20`) to trigger the flaky failure.
3. Observe that the assertion `expected: 2, actual: 1` fails intermittently at `process_instance_statistics_test.go:125`.

### Environment

_No response_

### Expected Behavior

_No response_

### PR DESCRIPTION
**Pull request by triskill**
…isticsMultiInstance/parallel_multi-instance_shows_body_tokens_not_scope_token
closes #490

### CODE REVIEW

File: Dockerfile.e2e [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM golang:1.25
```
[the remark this answers is not in the document]
Comment (triskill) [reply Dockerfile.e2e]: last GO version is 1.26, in project is used version 1.25

File: test/e2e/process_instance_statistics_test.go [hunk 2]
```diff
@@ -116,14 +117,15 @@ func TestGetProcessInstanceElementStatisticsMultiInstance(t *testing.T) {
...
+			require.NoError(t, err)
+			assert.Equal(t, http.StatusOK, resp.StatusCode())
+			require.NotNil(t, resp.JSON200)
 
-		activeByElement := collectActiveByElement(resp.JSON200)
-		assert.Equal(t, 2, activeByElement["Activity_0rae016"],
-			"should count child body tokens, not the parent scope token")
+			activeByElement := collectActiveByElement(resp.JSON200)
+			return activeByElement["Activity_0rae016"] == 2
+		}, 1*time.Second, 100*time.Millisecond, "should count child body tokens, not the parent scope token")
```
[the remark this answers is not in the document]
Comment (triskill) [reply test/e2e/process_instance_statistics_test.go]: i added conditions with return false and lastError variable to be logged when Eventually assert fails, instead of assertion inside Eventually.

File: test/e2e/process_instance_statistics_test.go [hunk 3]
```diff
@@ -110,20 +112,37 @@ func TestGetProcessInstanceElementStatisticsMultiInstance(t *testing.T) {
...
+			result := collectActiveByElement(resp.JSON200)
+			activeByElement = &result
+			return result["Activity_0rae016"] == testInputCollectionLen
+		}, 1*time.Second, 100*time.Millisecond, "should count child body tokens, not the parent scope token")
+		if !ok {
+			t.Logf("lastError=%v, activeTokenCount=%d, expectedTokenCount=%d",
+				lastError,
+				(*activeByElement)["Activity_0rae016"],
+				testInputCollectionLen)
+		}
```
[the remark this answers is not in the document]
Comment (triskill) [reply test/e2e/process_instance_statistics_test.go:146]: ok, i did it.

File: Dockerfile.e2e (hunk 1 above)
[the remark this answers is not in the document]
Comment (triskill) [reply Dockerfile.e2e]: current stable GO version is 1.26

**Comment by triskill** [comment]
Pleas suggest your opinion on adding `test-e2e-docker` Makefile option. It is for testing edge cases, when tests fails only in pipelines. I am not sure if we need it, i can drop it.

**Comment by Kuzna** [comment]
Hello, I don’t think it’s necessary to have this kind of test. To me, it looks like a duplicate because it’s already covered in the pipeline, so I don’t think it’s needed.

**Comment by triskill** [comment]
> Hello, I don’t think it’s necessary to have this kind of test. To me, it looks like a duplicate because it’s already covered in the pipeline, so I don’t think it’s needed.

@Kuzna Ok, i removed it.

**Review by Kuzna — APPROVED** [review]
